accepted/future-releases/0546-patterns/feature-specification.md

```diff
@@ -2809,24 +2809,44 @@ To match a pattern `p` against a value `v`:
         1.  Evaluate the key `expression` to `k` and call `containsKey(k)` on
```

Comment:
This part needs to be changed to just:

1. Evaluate the key `expression` to `k`.

(We shouldn't call `containsKey(k)` pre-emptively)